vm/devices/get/guest_emulation_transport/src/client.rs

```diff
@@ -465,6 +465,13 @@ impl GuestEmulationTransportClient {
         self.control.call(msg::Msg::FlushWrites, ()).await
     }
 
+    /// Report the fatal event to the host and flush the event queue. This ensures all the
```

Comment:
I would probably expand on this doc comment some more, adding in
- a intra-doc reference to the `event_log` method above 
- a brief explanation of when to use this API vs. the base event_log API

and similarly, I would update the `event_log` method docs to point out `event_log_fatal` exists, and when to use one vs. the other